tests/openebs-pool-container-failure_test.go

```diff
@@ -80,10 +82,45 @@ var _ = BeforeSuite(func() {
 var _ = Describe("BDD of openebs pool container failure experiment", func() {
 
 	// BDD TEST CASE 1
+	rv1 := 0
+	rc1 := 0
 	Context("Check for the components of openebs", func() {
 
 		It("Should check for creation of runner pod", func() {
 
+			//Getting the Sum of Resource Version before Chaos
+			csp_rv, err := client.CoreV1().Pods("openebs").List(metav1.ListOptions{LabelSelector: "app=cstor-pool"})
```

Comment:
Can you pls use a variable here, instead of hardcoded the labels `app=cstor-pool`

Reply:
Not really required now, as - it is not used in multiple places also it not getting changed. We can do that when we have any of the above requirements.